packages/@sanity/components/src/tooltip/tooltip.tsx

```diff
@@ -56,6 +58,13 @@ export function Tooltip(
       {
         name: 'offset',
         options: {offset: [0, 3]}
+      },
+      {
+        name: 'flip',
+        options: {
+          allowedAutoPlacements: props.allowedAutoPlacements || ['top', 'bottom'],
```

Comment:
Here we should remove the `|| ['top', 'bottom']` part. Otherwise it’s not possible to get the default setting without defining all possible placements?